templates/themes/app/pagelayout.html.twig

```diff
@@ -1,24 +1,32 @@
-{% extends '@ibexadesign/pagelayout_variables.html.twig' %}
+{% extends '@ibexadesign/pagelayout/variables.html.twig' %}
 
 {% block pagelayout_content %}
 <!DOCTYPE html>
 <html lang="{{ app.request.locale|replace({'_': '-'}) }}">
 <head>
     <meta charset="utf-8">
 
-    {% block page_head %}
```

Comment:
Same remark goes here for block names as with renames. Removing them would also make upgrades harder.

Reply:
As agreed f2f I'm skipping this